src/test/java/ai/rev/speechtotext/unit/RevAiJobTest.java

```diff
@@ -171,6 +173,42 @@ public void SubmitJobUrl_UrlAndOptionsAreSpecified_ReturnsARevAiJob() throws IOE
     assertRevAiJob(revAiJob, mockInProgressJob);
   }
 
+  @Test
+  public void SubmitJobUrl_UrlAndHumanTcOptionsSpecified_ReturnsARevAiJob() throws IOException {
```

Comment:
I'm not sure this needs a separate unit test. The SDK has no knowledge or validation for human vs non human options. We can just keep it to one test where we set all options

Reply:
that's a good point, I'll update the tests